### ISSUE TITLE
USER ISSUE: Cannot not load a single game

### ISSUE THREAD

**Issue by nas2803** [StrangeLoopGames/EcoIssues#6484]
Hello,

 I am just kicking out my single game and since this time, i cannot load or create a game.

When i select my game, i confirm by clicking on yes but nothing, no loading screan no error message.

I delete the folder AppData\LocalLow\Strange Loop Games\Eco but same problem.

[Crash AggregateException 03240217 - Copie.ecodmp.zip](https://github.com/StrangeLoopGames/EcoIssues/files/1845258/Crash.AggregateException.03240217.-.Copie.ecodmp.zip)

**Comment by MarjieAVolk** [comment]
So to clarify - There is just one specific save that doesn't load?

```
System.AggregateException: Une ou plusieurs erreurs se sont produites. ---> System.AggregateException: Une ou plusieurs erreurs se sont produites. ---> System.Reflection.TargetInvocationException: Une exception a été levée par la cible d'un appel. ---> System.Exception: Error(s) occurred when loading World
   à Eco.Core.Serialization.DataStore.LoadAsync[T](String folder, Action`2 createFunc)
   à Eco.World.PersistentChunkGrid.Load()
   à Eco.World.WorldPlugin..ctor()
```

**Comment by MarjieAVolk** [comment]
Also, what game version is this on?  You can check the game version in the lower left corner of the escape menu.

**Comment by simplay** [comment]
The same happens to me. I don't have any prior save-games since I recently bought this game. 

Started the game, Clicked on **New** , then I entered some random world name, checked **allow friends to join** and **generate new** and finally pressed **start**. After that, a dialogue-box popped up "Generate a new world? This will take a while.". I clicked onto "yes" but then nothing happens. Waited for like 15 minutes.

I'm running this game on Ubuntu 16.04 on decent hardware. Are there any additional dependencies I should have installed? 

Is there some debug mode I could run to provide you some more information?
